R/init.R

```diff
@@ -34,11 +34,11 @@
 #'   The header of the app.
 #' @param footer (`shiny.tag` or `character(1)`)
 #'   The footer of the app.
-#' @param id (`character`)
-#'   module id to embed it, if provided,
-#'   the server function must be called with [shiny::moduleServer()];
-#'   See the vignette for an example. However, [ui_teal_with_splash()]
-#'   is then preferred to this function.
+#' @param id optional (`character`)
+#'   specifying the module id to be used when embedding the teal app within another `shiny` app as a module.
+#'   - If an id is provided, the server function must be called using [`shiny::moduleServer()`] with the specified id.
+#'   However, for better integration use [`ui_teal_with_splash()`] and [`srv_teal_with_splash()`].
+#'   - If no id is given (i.e., `id = character(0)`), the teal app runs as a standalone application without submodule integration.
```

Comment:
This part it too much for argument description, please add "see `Details` and move it there.